pydoctor/templatewriter/writer.py

```diff
@@ -82,22 +82,30 @@ def writeIndividualFiles(self, obs: Iterable[model.Documentable]) -> None:
 
     def writeSummaryPages(self, system: model.System) -> None:
         import time
-        for pclass in summary.summarypages:
+        for pclass in summary.summaryPages(system):
             system.msg('html', 'starting ' + pclass.__name__ + ' ...', nonl=True)
             T = time.time()
             page = pclass(system=system, template_lookup=self.template_lookup)
             with self.build_directory.joinpath(pclass.filename).open('wb') as fobj:
                 flattenToFile(fobj, page)
             system.msg('html', "took %fs"%(time.time() - T), wantsnl=False)
 
+        if len(system.root_names) == 1:
+            # If there is just a single root module it is written to index.html to produce nicer URLs.
+            # To not break old links we also create a symlink from the full module name to the index.html
+            # file. This is also good for consistency: every module is accessible by <full module name>.html
+            root_module_path = (self.build_directory / (list(system.root_names)[0] + '.html'))
+            root_module_path.unlink()
+            root_module_path.symlink_to('index.html')
```

Comment:
I just added a check to create a symlink so that old links outside of pydoctor don't break. I initially tried adding the check in driver but then the symlink creation fails because some tests use an `InMemoryWriter`. So having the check in the `TemplateWriter` class makes more sense.